packages/ui-components/src/lib/theme/ThemeBase.tsx

```diff
@@ -7,7 +7,6 @@ const muiTheme = createTheme();
 
 export interface ThemeBaseProps {
   as?: keyof JSX.IntrinsicElements;
-  children: ReactNode;
```

Comment:
We don't need this, because `FC` includes the type for `children` (via `PropsWithChildren`) by default.